pkg/yurtctl/cmd/convert/convert.go

```diff
@@ -216,6 +218,12 @@ func (co *ConvertOptions) Complete(flags *pflag.FlagSet) error {
 	}
 	co.YurthubHealthCheckTimeout = yurthubHealthCheckTimeout
 
+	waitServantJobTimeout, err := flags.GetDuration("wait-servant-job-timeout")
+	if err != nil {
+		return err
+	}
+	kubeutil.WaitServantJobTimeout = waitServantJobTimeout
```

Comment:
I think we should not modify var `kubeutil.WaitServantJobTimeout`. how about add a `WaitServantJobTimeout` field for `ConvertOptions`?  and then use `co.WaitServantJobTimeout` for timeout of `RunServantJobs`.

Reply:
down